AAT_android/app/src/main/java/com/ase/aat_android/data/Course.java

```diff
@@ -0,0 +1,47 @@
+package com.ase.aat_android.data;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+/**
+ * Wrapper class for datastore Course
+ * Created by anahitik on 02.02.17.
+ */
+
+public class Course implements Serializable{
+    com.aat.datastore.Course course;
```

Comment:
This class is used when passing Course object from CoursesActivity to CourseActivity.
Before all fields of datastore Course object were put into the extras (as datastore Course is not implementing Serializable, it is not possible to put its object into an extra).
This class provides easier way to Course object.